<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_principal</class>
 <widget class="QMainWindow" name="frm_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caixa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Receita Total</string>
    </property>
    <widget class="QLabel" name="label_receitaTotal">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string> Laçamentos:</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_excluir">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Excluir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_editar">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Editar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_lancar">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Lançar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Despesa Total</string>
    </property>
    <widget class="QLabel" name="label_despesaTotal">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Saldo Atual</string>
    </property>
    <widget class="QLabel" name="label_saldoTotal">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCaixas">
    <property name="title">
     <string>Caixas</string>
    </property>
    <addaction name="actionLan_amentos"/>
   </widget>
   <addaction name="menuCaixas"/>
  </widget>
  <action name="actionLan_amentos">
   <property name="text">
    <string>Lançamentos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
